### PR TITLE
feat(route): add `/autocentre`

### PR DESCRIPTION
**Pull request by driversti**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/autocentre
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

_Автоцентр_ - Automotive website N1 in Ukraine

### CODE REVIEW

File: lib/routes/autocentre/index.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { Data, Route } from '@/types';
+import parser from '@/utils/rss-parser';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '',
```
Comment (TonyRL) [inline lib/routes/autocentre/index.ts:6]: path should start with `/`

File: lib/routes/autocentre/index.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    name: 'Автоцентр.ua',
```
Comment (TonyRL) [inline lib/routes/autocentre/index.ts:7]: name should not be the same as namespace's name

File: lib/routes/autocentre/index.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    maintainers: ['driversti'],
+    example: 'autocentre',
```
Comment (TonyRL) [inline lib/routes/autocentre/index.ts:9]: example should start with `/`

File: lib/routes/autocentre/index.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+async function handler(): Promise<Data> {
+    const feed = await parser.parseURL('https://autocentre.ua/rss');
+
+    return {
+        title: feed.title as string,
+        link: feed.link,
+        description: feed.description,
+        language: 'uk',
+        date: feed.lastBuildDate,
```
Comment (TonyRL) [inline lib/routes/autocentre/index.ts:29]:
```suggestion
```

File: lib/routes/autocentre/index.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    handler,
+};
+
+const createItem = (item) => ({
+    title: item.title,
+    link: item.link,
+    description: item.contentSnippet,
+});
+
+const getFromCacheOrNew = (item) => cache.tryGet(item.link, () => createItem(item));
```
Comment (TonyRL) [inline lib/routes/autocentre/index.ts:19]: Don't use cache if it doesn't involve additional requests

File: lib/routes/autocentre/index.ts (hunk 2 above)
Comment (driversti) [reply lib/routes/autocentre/index.ts:7]: Could you please suggest what I should put here? I haven't noticed it in the Documentation.

File: lib/routes/autocentre/index.ts (hunk 3 above)
Comment (driversti) [reply lib/routes/autocentre/index.ts:9]: What feed category should I choose? I didn't find any category related to automotive. Can we add one?

Comment (TonyRL) [reply lib/routes/autocentre/index.ts]: new-media

File: lib/routes/autocentre/index.ts (hunk 2 above)
Comment (driversti) [reply lib/routes/autocentre/index.ts]: Done

File: lib/routes/autocentre/index.ts (hunk 3 above)
Comment (driversti) [reply lib/routes/autocentre/index.ts]: Done

File: lib/routes/autocentre/index.ts (hunk 4 above)
Comment (driversti) [reply lib/routes/autocentre/index.ts]: Done

File: lib/routes/autocentre/index.ts (hunk 5 above)
Comment (driversti) [reply lib/routes/autocentre/index.ts]: Done

File: lib/routes/autocentre/index.ts (hunk 1 above)
Comment (driversti) [reply lib/routes/autocentre/index.ts]: Done

File: lib/routes/autocentre/index.ts [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+};
+
+const createItem = (item) => ({
+    title: item.title,
+    link: item.link,
+    description: item.contentSnippet,
+});
+
+async function handler(): Promise<Data> {
+    const feed = await parser.parseURL('https://autocentre.ua/rss');
```
Comment (TonyRL) [inline lib/routes/autocentre/index.ts:20]: You can use the redirected URL
```suggestion
    const feed = await parser.parseURL('https://www.autocentre.ua/rss');
```

**Pull request merged**